checks/raw/fuzzing.go

```diff
@@ -87,6 +89,30 @@ var languageFuzzSpecs = map[clients.LanguageName]languageFuzzConfig{
 			"Property-based testing in Haskell generates test instances randomly or exhaustively " +
 				"and test that specific properties are satisfied."),
 	},
+	// Fuzz patterns for JavaScript and TypeScript based on property-based testing.
+	//
+	// Based on the import of one of these packages:
+	// * https://fast-check.dev/
+	//
+	// This is not an exhaustive list.
+	clients.JavaScript: {
+		filePattern: "*.js",
```

Comment:
Is there a simple way to also check for extensions such as cjs, mjs, jsx,cjsx, mjsx which are some alternate extensions for JavaScript code? Same need for TypeScript. Testing js only covers must of the cases, but adding these extra ones might extend the coverage a little bit more.